src/test/java/hygieia/transformer/JunitXmlToTestCapabilityTransformerTest.java

```diff
@@ -0,0 +1,47 @@
+
+package hygieia.transformer;
+
+import com.capitalone.dashboard.model.TestCapability;
+import com.capitalone.dashboard.model.TestSuite;
+import com.capitalone.dashboard.model.quality.JunitXmlReport;
+import org.junit.Test;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.Unmarshaller;
+
+import java.io.File;
+import java.io.FileReader;
+import java.net.URL;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+public class JunitXmlToTestCapabilityTransformerTest {
+
+    JunitXmlToTestCapabilityTransformer sut = new JunitXmlToTestCapabilityTransformer();
+
+    @Test
+    public void producesJunitTestResult() throws Exception {
+
+        URL url = getClass().getResource("/junit.xml");
+        File file = new File(url.getPath());
+        FileReader fileReader = new FileReader(file);
+        JunitXmlReport responseReport = null;
+        JAXBContext jaxbContext = JAXBContext.newInstance(JunitXmlReport.class);
```

Comment:
JAXBContext should be static and should not be initialized everytime.

Reply:
even though it is unit test and is executed once in future if we add more tests it will be useful.